src/main/java/core/basesyntax/util/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.util;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_QUANTITY = Color.values().length;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(COLOR_QUANTITY);
```

Comment:
```suggestion
        int index = random.nextInt(colors.length);
        return colors[index].name();
```

see the [link](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)